apps/backend/src/drink-action/entities/drink-action.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { IsNumber, IsUUID } from 'class-validator';
+
+export class DrinkAction {
+  @IsUUID()
+  id: string;
+
+  @IsUUID()
+  drinkId: string;
+
+  @IsUUID()
+  eventId: string;
```

Comment:
It might be beneficial to add Swagger comments to these fields, but it's not completely necessary.

```suggestion
  /**
   * @example "123e4567-e89b-12d3-a456-426614174000"
   */
  @IsUUID()
  id: string;

  /**
   * Id of the drink involved in the action
   * @example "123e4567-e89b-12d3-a456-426614174001"
   */
  @IsUUID()
  drinkId: string;

  /**
   * Id of the event where the drink action happened
   * @example "123e4567-e89b-12d3-a456-426614174002"
   */
  @IsUUID()
  eventId: string;
```